worldedit/common.lua

```diff
@@ -109,6 +109,5 @@ function mh.finish(manip, data)
 	-- Update map
 	manip:set_data(data)
 	manip:write_to_map()
-	manip:update_map()
```

Comment:
heh, no

worldedit should still support previous versions where this function call is required.

Reply:
https://github.com/minetest/minetest/commit/ab371cc93491baf0973ecc94b96c3a1fdb4abfd5
It's in the 0.4.16 release. People who use outdated stable minetest likely also use outdated worldedit l think.

Reply:
Not necessarily.